src/target.rs

```diff
@@ -158,17 +158,23 @@ impl Target {
                 let release = info.release().replace(".", "_").replace("-", "_");
                 format!("openbsd_{}_amd64", release)
             }
-            (Os::Linux, _) => format!("{}_{}", manylinux, self.arch),
+            (Os::Linux, _) => {
+                let mut tags = vec![format!("{}_{}", manylinux, self.arch)];
+                for alias in manylinux.aliases() {
+                    tags.push(format!("{}_{}", alias, self.arch));
+                }
+                tags.join(".")
+            }
             (Os::Macos, Arch::X86_64) => {
                 if universal2 {
-                    "macosx_10_9_universal2".to_string()
+                    "macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2".to_string()
```

Comment:
Same as https://github.com/PyO3/maturin/issues/523#issuecomment-833517235: What extra information do we get through this that we don't get through `macosx_10_9_universal2`?

Reply:
Old version of `pip` which doesn't know about `universal2` can still pick up this wheel.

Reply:
I'd prefer not to support old versions of pip and instead tell people to update their pip, given that the pip team doesn't support old versions and they print a yellow warning after every command run from an outdated version